### ISSUE TITLE
The 'for' loop in the settle mainer() function runs only once - return is inside the loop block

### ISSUE THREAD

**Issue by MartinCupak** [ADACS-Australia/beans#8]
* Operating System: All

### Description

The 'for' loop in the mainer() function runs only once - return is inside the loop block.
Is that intended behaviour?

Line numbers for reference, it is module setlle.cc

```
85  int mainer(double* flu, double* Z, double* X, double* mdo, int* docomp,
86         double* trec, double* alpha, double* fluen, double* radius, double*mass)

141  for (int ii=1; ii<=1; ii++) {

276  return 0;
277  } // loop through flux
```

**Comment by MartinCupak** [comment]
Note: this was introduced in commit f2b5f099859c8fc92ea5ce3c4b2ce1f58cfc77e2

**Comment by MartinCupak** [comment]
closed with commit e5c7d6e
